docs/projects/speedtest-exporter.md

```diff
@@ -142,7 +142,7 @@ Real example where the **tests** will be done **every hour**:
   scrape_interval: 1h
   scrape_timeout: 1m
   static_configs:
-    - targets: ['speedtest-exporter:9798']
+    - targets: ['localhost:9798']
```

Comment:
Should we explain here, that this `localhost` is the address from where the exporter is running?